raster.py

```diff
@@ -70,13 +75,19 @@ def get_animations_path(filename: str) -> str:
   return f"{root}{ANIMATIONS_BASE}{filename}"
 
 def mount_gdrive():
-  global MOUNTED
-  if not MOUNTED:
-    MOUNTED = True
+  if not os.path.exists(GDRIVE_BASE):
```

Comment:
I'm guessing you're checking if gdrive is already mounted? If so, can you add a comment explaining that? It is not immediately obvious

Reply:
this is unrelated to this PR and only exists because a previous PR hasnt been merged yet

Reply:
unrelated to this change.  i'm waiting for my other PR to get approval.